_posts/2018-11-11-kanban.md

```diff
@@ -0,0 +1,124 @@
+
+Last year I was part of the effort to get the [Puppet Development Kit](https://puppet.com/blog/develop-modules-faster-new-puppet-development-kit) built and shipped. Since I'm very interested in team effectiveness and agility, at one point I took a day to re-organize our kanban boards to streamline the info exposed and reduce the friction when using them. Only after leaving the team, I realised [the value of having a low-friction JIRA configuration](https://twitter.com/nigelkersten/status/1024675669429518336):
+
+![a screenshot of the tweet linked above](/log/assets/2018-11-11-kanban/tweet.png)
+
+Thanks due to some prodding by [@sigje](https://twitter.com/sigje/status/1025121668191023104) and some internal folks, I've finally come around to writing this up.
+
+![a screenshot of the tweet linked above](/log/assets/2018-11-11-kanban/prod.png)
+
+I'll start out with an overview of how the boards look like today, and will have the life of a ticket in the second half.
+
+# Dual-track Agile
+
+One of my big concerns last year was a big backlog of things we wanted to achieve with the PDK, but were unable to address at "this" point in time. With the default setup of JIRA (back then), this was loading up the board with tickets that were not relevant to the day-to-day activities. At the same time, this glut of tickets obscured the view of what should be coming next, as individual stories were too detailed, and not high-lighted.
+
+At the time I stumbled over some folks talking about dual-track agile: splitting design and long-term planning out from the day-to-day development board. This allowed me to represent the high-level planning and progress on one board, and to declutter the day-to-day detail view necessary for development on the primary board.
+
+# The "State of Play" Board
+
+The main board for day-to-day development usage should look familiar to everyone who had to work with JIRA.
+
+![the main board](/log/assets/2018-11-11-kanban/state_of_play_board.png)
+
+We were already used to using Epics to group tickets, so I followed JIRA's default of Epics as swimlanes. At that time we were using our internal JIRA defaults for board columns that was built for the much larger team and project of Puppet Enterprise, which had something like 8 different ticket states to handle Dev/QA/CI handoffs. In the PDK team we were mostly using [Travis CI](https://travis-ci.com) to directly test pull requests and code would go straight to master after a review from a second engineer. This resulted in the six columns that we have now: Backlog, Groomed, Blocked, In Progress, Review, and Done.
+
+Astute readers might notice that the backlog column doesn't show up on the screenshot. This is a recent change in JIRA that I really love. The first column of the board is not shown on the main board anymore, but moved to a separate - aptly named "backlog" - view.
```

Comment:
I'm curious what this backlog view looks like and how you interact with it.

Reply:
I'll add a screenshot.